jib-cli/src/main/java/com/google/cloud/tools/jib/cli/cli2/Build.java

```diff
@@ -40,18 +43,29 @@ public Integer call() {
     try {
       ConsoleLogger logger =
           CliLogger.newLogger(globalOptions.getVerbosity(), globalOptions.getConsoleOutput());
+
+      Path buildFile = globalOptions.getBuildFile();
+      if (!Files.isReadable(buildFile)) {
+        logger.log(Level.ERROR, "Cannot open Build File YAML: " + buildFile);
```

Comment:
Maybe we can make the error message a bit more specific - "The Build File YAML either does not exist or cannot be opened for reading" since `isReadable` checks for both.